src/utils/sort-errors/sortErrors.js

```diff
@@ -0,0 +1,12 @@
+export const sortErrors = (errors, desiredOrder) => {
+  var orderedKeys = [];
+  const errorKeys = Object.keys(errors);
+  desiredOrder.forEach((key) => {
+    errorKeys.forEach((errorKey) => {
```

Comment:
This is me just nit picking here really, but I think a nested loop with what looks like n^2 complexity can be avoided here if we were to just convert the errorKeys to a set and then do a errorKeys.has() with the desiredOrder key removing the need for the nested loop. 

In reality it's unlikely to make much of a performance difference with the small number of errors you'd expect to have. 

So i'm not asking for a change or anything just thought it was worth noting.

Reply:
I've done this now